loama/src/components/LoButton.vue

```diff
@@ -48,16 +50,30 @@ button {
     border-radius: var(--base-corner);
     height: fit-content;
     border: none;
+    cursor: pointer;
+    border: .2em solid;
```

Comment:
`em` can act funky when it comes to screen scaling, because it's based on the font-size of the targeted element. If you want some more consistency, use `rem`
```suggestion
    border: 0.25rem solid;
```